### PR TITLE
add migrations for new mandatees -Somers +Rutten

### PR DESCRIPTION
**Pull request by MikiDi**

### CODE REVIEW

File: config/migrations/20231108114709-new-minister-data.ttl [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    ns2:uuid "ccd0aa72-bad7-48a8-b260-fee95f17a8fd" ;
+    dcterms:title "Vlaams minister van Binnenlands Bestuur, Bestuurszaken, Inburgering en Gelijke Kansen" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/d1290ae5-9a25-4e53-924e-dcfd76cf0886> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0697> ;
+    ns1:rangorde 4 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "d1290ae5-9a25-4e53-924e-dcfd76cf0886" ;
+    dcterms:title "Vlaams minister van Onderwijs, Sport, Dierenwelzijn en Vlaamse Rand" ;
```
Comment (ValenberghsSven) [inline config/migrations/20231108114709-new-minister-data.ttl:98]: viceminister needs no title (none of the others have one)

File: config/migrations/20231108114709-new-minister-data.ttl [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    dcterms:title "Minister-president van de Vlaamse Regering" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0702> .
+
+<http://themis.vlaanderen.be/id/mandatee/f6189ea6-eb79-459e-96b9-261eaa960c1e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907de6670526694a061b> ;
+    ns1:rangorde 2 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "f6189ea6-eb79-459e-96b9-261eaa960c1e" ;
+    dcterms:title "Vlaams minister van Economie, Innovatie, Werk, Sociale economie en Landbouw" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
```
Comment (sergiofenoll) [inline config/migrations/20231108114709-new-minister-data.ttl:123]: Is this correct? "Vlaams minister van Economie, Innovatie, Werk, Sociale economie en Landbouw" is Jo Brouns according to [this](https://www.vlaanderen.be/vlaamse-regering), so rangorde should be 9 not 2.

But this also points to Hilde Crevits (persoon themis links resolved to her). Crevits mandataris title should be "Vlaams minister van Welzijn, Volksgezondheid en Gezin", but then we're still missing Brouns

File: config/migrations/20231108114709-new-minister-data.ttl [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "25e60e01-f45b-4c55-9e04-1b8f1f1e5417" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0703> .
+
+<http://themis.vlaanderen.be/id/mandatee/671967e1-ad8f-45ad-a1e4-1c20ed5c534e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0718> ;
+    ns1:rangorde 6 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "671967e1-ad8f-45ad-a1e4-1c20ed5c534e" ;
+    dcterms:title "Vlaams minister van Welzijn, Volksgezondheid, Gezin en Armoedebestrijding" ;
```
Comment (sergiofenoll) [inline config/migrations/20231108114709-new-minister-data.ttl:59]: Not a mandataris title that I can find here, mistake or out-of-date page? https://www.vlaanderen.be/vlaamse-regering

Comment (sergiofenoll) [reply config/migrations/20231108114709-new-minister-data.ttl:59]: Mandataris with rangorde 6 should have title "Vlaams minister van Financiën en Begroting, Wonen en Onroerend Erfgoed". This title appears later as rangorde 7, which breaks the composition for later ministers

File: config/migrations/20231108114709-new-minister-data.ttl [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+<http://themis.vlaanderen.be/id/mandatee/25e60e01-f45b-4c55-9e04-1b8f1f1e5417> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907de6670526694a061b> ;
+    ns1:rangorde 2 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "25e60e01-f45b-4c55-9e04-1b8f1f1e5417" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0703> .
+
+<http://themis.vlaanderen.be/id/mandatee/671967e1-ad8f-45ad-a1e4-1c20ed5c534e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0718> ;
```
Comment (sergiofenoll) [inline config/migrations/20231108114709-new-minister-data.ttl:55]: This person resolves to Wouter Beke.

**Pull request closed without merging**